src/features/Applet/Schedule/Calendar/EventContainerWrapper/EventContainerWrapper.tsx

```diff
@@ -0,0 +1,93 @@
+import { useEffect, useRef } from 'react';
+import { Box } from '@mui/material';
+
+import { CalendarViews } from '../Calendar.types';
+import { EventContainerWrapperProps, EventsStartEndDates } from './EventContainerWrapper.types';
+import {
+  getVariables,
+  getEventClassNames,
+  getOverlappingEvents,
+} from './EventContainerWrapper.utils';
+
+export const EventContainerWrapper = ({
+  children,
+  events,
+  components,
+}: EventContainerWrapperProps) => {
+  const wrapperRef = useRef<HTMLElement>();
+  const activeView = components?.activeView;
+  const isWeekView = activeView === CalendarViews.Week;
+
+  useEffect(() => {
+    const {
+      VISIBLE_EVENTS_QUANTITY,
+      ALL_COL_QUANTITY,
+      INDEX_SHOW_MORE_BTN,
+      LEFT_OFFSET_COEFFICIENT,
+      LEFT_OFFSET_START_VAL,
+      VALUE_DECREASING_FULL_CONTAINER_WIDTH,
+    } = getVariables(isWeekView);
+
+    setTimeout(() => {
```

Comment:
is no chance without timeout?

Reply:
I've tried to use async await in different versions, but it didn't work